tests/test_pandas.py

```diff
@@ -152,6 +155,17 @@ def test_decide_from_contexts_df():
 
   assert_equal(output["output"]["b"]["predicted_value"], "Pierre")
 
+@with_setup(setup_complex_agent_with_data, teardown)
+def test_decide_from_contexts_df_zero_rows():
+  tree = CLIENT.get_decision_tree(AGENT_ID, COMPLEX_AGENT_DATA.last_valid_index().value // 10 ** 9)
+  test_df = COMPLEX_AGENT_DATA.iloc[:0, :]
```

Comment:
```suggestion
  test_df = pd.Dataframe([])
```
What about just using something like this ?

Reply:
It's doesn't test the same thing ! The current test_df has some columns whereas the suggested one doesn't.

Both could be interesting to test but I am not sure about the expected behavior of the function on the suggested dataFrame.

Reply:
Ah right ! Let's leave in like this then